core/chains/evm/config/toml/defaults/Arbitrum_Mainnet.toml

```diff
@@ -1,6 +1,7 @@
 # Arbitrum is an L2 chain. Pending proper L2 support, for now we rely on their sequencer
 ChainID = '42161'
 ChainType = 'arbitrum'
+FinalityDepth = 10
```

Comment:
This seems wrong.
Arbitrum needs to have a large finalityDepth, or use FinalityTags.

Reply:
With `FinalityTagBypass=true` Headtracker fallbacks to FinalityDepth. TXM does not fully support FT even if `FinalityTagBypass=false`. We should try to avoid unnecessary reduction of FinalityDepth until FinalityTagBypass is removed and Juan finishes TXM changes.

Reply:
Hm, large finality depth (>1000) might cause spike in the CPU usage due to HeadTracker's inefficient memory usage. We might want to wait with merging until [this](https://smartcontract-it.atlassian.net/browse/BCI-3398) work is done (ETA is end of August).